main.yml

```diff
@@ -766,6 +764,66 @@ service-registry:
         title: Documentation
         navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry
 
+
+## App Services (RHOAS) APIs
+app-services-apis:
```

Comment:
We want apis grouped and separate from frontends (they do not specifically match 1:1 with frontends) that is why I have created separate section instead of adding to the existing objects